docs/source/index.rst

```diff
@@ -18,26 +18,31 @@ Welcome to SkyPilot!
    </p>
 
    <p style="text-align:center">
-   <strong>Run jobs on any cloud, easily and cost effectively</strong>
+   <strong>Run LLMs and AI on Any Cloud</strong>
    </p>
 
-SkyPilot is a framework for easily and cost effectively running ML workloads on any cloud.
+SkyPilot is a framework for running LLMs, AI, and batch jobs on any cloud, offering maximum cost savings, highest GPU availability, and managed execution.
 
-SkyPilot abstracts away cloud infra burden:
+SkyPilot **abstracts away cloud infra burdens**:
 
-- Launch jobs & clusters on any cloud (AWS, Azure, GCP, Lambda Cloud)
-- Find scarce resources across zones/regions/clouds
-- Queue jobs & use cloud object stores
+- Launch jobs & clusters on any cloud
+- Easy scale-out: queue and run many jobs, automatically managed
+- Easy access to object stores (S3, GCS, R2)
 
-SkyPilot cuts your cloud costs:
+SkyPilot **maximizes GPU availability for your jobs**:
 
-* :ref:`Managed Spot <spot-jobs>`: **3x cost savings** using spot VMs, with auto-recovery from preemptions
-* :ref:`Autostop <auto-stop>`: hands-free cleanup of idle clusters
-* :ref:`Benchmark <benchmark-overview>`: find best VM types for your jobs
-* Optimizer: **2x cost savings** by auto-picking best prices across zones/regions/clouds
+* Provision in all zones/regions/clouds you have access to (`the Sky <https://arxiv.org/abs/2205.07147>`_), with automatic failover
```

Comment:
very nit: italicize "Sky"?

Reply:
I tried
```
`the *Sky* <https://arxiv.org/abs/2205.07147>`_
*`the Sky <https://arxiv.org/abs/2205.07147>`*_
*`the Sky <https://arxiv.org/abs/2205.07147>`_*
```
None worked. There doesn't appear to be a way to nest things in rST.